README.md

```diff
@@ -20,7 +20,11 @@ Add the following dependency to your `build.gradle`
 implementation 'com.grafana:grafana-opentelemetry-starter:1.0.1'
 ```
 
-Next, register the OpenTelemetry logback appender in `logback-spring.xml` (or `logback.xml`):
+The next step depends on your logging framework - logback and log4j2 are supported: 
+
+## Logback
+
+If you use logback, register the OpenTelemetry logback appender in `logback-spring.xml` (or `logback.xml`):
```

Comment:
Let's include that a logging appender file is typically added under a project's `main/resources` directory.